### PR TITLE
Endpoint to receive from ADA

### PR DESCRIPTION
**Pull request by DevChima**

### CODE REVIEW

File: ada/tasks.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from django.conf import settings
+from requests.exceptions import RequestException
+from temba_client.exceptions import TembaHttpError
+from temba_client.v2 import TembaClient
+
+from ndoh_hub.celery import app
+
+
+rapidpro = None
+if settings.ADA_RAPIDPRO_URL and settings.ADA_RAPIDPRO_TOKEN:
```
Comment (erikh360) [inline ada/tasks.py]: Does ADA have its own rapidpro org? not on momconnect?

File: ada/views.py [hunk 2]
```diff
@@ -2,8 +2,23 @@
...
 from .models import RedirectUrl, RedirectUrlsEntry
+from .tasks import submit_whatsappid_to_rapidpro
+
+
+class RapidProStartFlowView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request: HttpRequest) -> HttpResponse:
+        whatsappid = request.query_params["whatsappid"]
+        submit_whatsappid_to_rapidpro(whatsappid)
```
Comment (erikh360) [inline ada/views.py]: This would need to be `.delay()` so that it doesn't happen in request and rather goes to the celery queue

Comment (erikh360) [reply ada/views.py]: ...and maybe add a test for the view with a patch on the task, then task.delay.assertCalledWith...

File: ndoh_hub/settings.py [hunk 3]
```diff
@@ -51,8 +51,13 @@
     "django.contrib.staticfiles",
     # 3rd party
     "raven.contrib.django.raven_compat",
-    "rest_framework",
     "rest_framework.authtoken",
+    "rest_auth",
```
Comment (erikh360) [inline ndoh_hub/settings.py]: Are all these auth things needed? Can't we do token auth?

**Review by erikh360 — COMMENTED** [review]
few comments and questions

File: ada/tasks.py (hunk 1 above)
Comment (DevChima) [reply ada/tasks.py]: No it doesn't. I wasn't sure this was so I thought it was just a project reference thing. I'll remove

File: ndoh_hub/settings.py (hunk 3 above)
Comment (DevChima) [reply ndoh_hub/settings.py]: Token should be fine. I'll remove rest_auth and the url

File: ndoh_hub/settings.py [hunk 4]
```diff
@@ -51,8 +51,13 @@
     "django.contrib.staticfiles",
     # 3rd party
     "raven.contrib.django.raven_compat",
-    "rest_framework",
     "rest_framework.authtoken",
+    "rest_framework",
+    # "django.contrib.sites",
```
Comment (erikh360) [inline ndoh_hub/settings.py]: we can remove these completely

**Review by erikh360 — APPROVED** [review]
just the one comment

**Pull request merged**